xknx/knxip/tunnelling_request.py

```diff
@@ -31,14 +32,14 @@ def __init__(
         xknx: XKNX,
         communication_channel_id: int = 1,
         sequence_counter: int = 0,
-        cemi: CEMIFrame | None = None,
+        cemi: CEMIFrame | TPDU | None = None,
     ):
         """Initialize TunnellingRequest object."""
         super().__init__(xknx)
 
         self.communication_channel_id = communication_channel_id
         self.sequence_counter = sequence_counter
-        self.cemi: CEMIFrame | None = (
+        self.cemi: CEMIFrame | TPDU | None = (
```

Comment:
This is now a pretty misleading attribute name 😬
Is a CemiFrame a subtype of TPDU (in the protocol, not in xknx)?